### PR TITLE
Feature/comparison tad tools

### PR DESCRIPTION
**Pull request by laurensWe**
This Draft PR completes the checklist for TAD tools for specifically AiVerify, this Draft PR is created to collect feedback on the way to go for the other three tools that are up for research.

### CODE REVIEW

File: docs/Projects/TAD/tools_comparison.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+|                          | Help & Documentation    | 2.8   |
+|                          | Performance Efficiency  | 9     |
+|                          | Maintainability         | 15.8  |
+|                          | Security                | 8.3   |
+|                          | Compatibility           | 10.4  |
+|                          | Accessibility           | 0     |
+|                          | Portability             | 7.5   |
+|                          | Deployment              | 1.5   |
+|                          | Legal & Compliance      | 19    |
+| VerifyML                 | Functionality           |       |
```
Comment (anneschuth) [inline docs/Projects/TAD/tools_comparison.md:24]: Couldn't we make VerifyML a second column (next to the one already filled for AIVerify)?

Comment (anneschuth) [reply docs/Projects/TAD/tools_comparison.md:24]: I was hoping we could do that also for the larger tables below

Comment (laurensWe) [reply docs/Projects/TAD/tools_comparison.md:24]: Then the possibility for comments would not be there though, and i think that is more valuable then a specific number

File: docs/Projects/TAD/tools_comparison.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
+# Comparison of tools for transparency of algorithmic decision making
+
+In [tools.md](tools.md) we have researched a few tools which we want to investigate further, this document is the next
+step in that investigation. The checklist we will lay these tools against is written down at [index.md](index.md). The
+Fulfilled column will give a numerical value based on the priority which can then be compared with the other tools per
+section. The scoring is as follows: {M:4, S:3, C:2, W:1}.
```
Comment (berrydenhartog) [inline docs/Projects/TAD/tools_comparison.md:6]: i would make W -1 or -2. The W means it should not be in the product. 
```suggestion
section. The scoring is as follows: {M:4, S:3, C:2, W:-1}.
```

File: docs/Projects/TAD/tools_comparison.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+total_score = 8.3
+
+### Compatibility
+
+| Requirement                                                                                                                | Priority | Fulfilled | Comments                                                                   |
+|:---------------------------------------------------------------------------------------------------------------------------|:---------|-----------|----------------------------------------------------------------------------|
+| The tool is compatible with existing systems and infrastructure.                                                           | M        | 1         | As it is a container it can run on Kubernetes and therefore at Digilab     |
+| The tool supports industry-standard data formats and protocols.                                                            | M        | 1         | Most Datasets and Models are supported by the tool                         |
+| The tool operates seamlessly on supported operating systems and hardware platforms.                                        | S        | 0.3       | On Windows and Mac it does not run well out of box                         |
```
Comment (berrydenhartog) [inline docs/Projects/TAD/tools_comparison.md:171]: If it is containerized it should be able to run everywhere

**Review by berrydenhartog — COMMENTED** [review]
Added some small comments

File: docs/Projects/TAD/tools_comparison.md (hunk 2 above)
Comment (laurensWe) [reply docs/Projects/TAD/tools_comparison.md:6]: Currently we wouldn't ;) have any would not haves in our checklist but I agree with you here about the scoring and will incorporate it

File: docs/Projects/TAD/tools_comparison.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+| Deployment             | 1.5            | 0.6            | 1.2                            | 3.6              |                          |
+| Legal & Compliance     | 19             | 16             | 16                             | 16               |                          |
+| **Total**              | **133.3**      | **120.1**      | **120.7**                      | **108.2**        | ****                     |
+
+The scores are calculated in these individual markdowns
+[aiverify_checklist.md](aiverify_checklist.md),
+[verifyml_checklist.md](verifyml_checklist.md),
+[ibm_360_research_toolkit_checklist.md](ibm_360_research_toolkit_checklist.md),
+[holisticai_checklist.md](holisticai_checklist.md),
+[ai_assessment_tool_checklist.md](ai_assessment_tool_checklist.md).
```
Comment (anneschuth) [inline docs/Projects/TAD/tools_comparison.md:33]: Moved this to above

File: docs/Projects/TAD/tools_comparison.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+In [tools.md](tools.md) we have researched a few tools which we want to
+investigate further, this document is the next step in that investigation.
+The checklist we will lay these tools against is written down at
+[index.md](index.md). The Fulfilled column will give a numerical value based
+on the priority which can then be compared with the other tools per section.
+The scoring is as follows: {M:4, S:3, C:2, W:-1}.
+
+## Summary of the comparison
+
+| Requirement            | AIVerify Score | VerifyML Score | IBM 360 Research Toolkit Score | Holisticai Score | AI Assessment Tool Score |
```
Comment (anneschuth) [inline docs/Projects/TAD/tools_comparison.md:12]:
```suggestion
| Requirement            | [AIVerify](aiverify_checklist.md) | [VerifyML](verifyml_checklist.md) | [IBM 360 Research Toolkit](ibm_360_research_toolkit_checklist.md) | [Holisticai](holisticai_checklist.md) | [AI Assessment Tool](ai_assessment_tool_checklist.md) |
```

File: docs/Projects/TAD/tools_comparison.md (hunk 4 above)
Comment (anneschuth) [inline docs/Projects/TAD/tools_comparison.md:33]:
```suggestion
```
Moved this to above

File: docs/Projects/TAD/tools_comparison.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
+# Comparison of tools for transparency of algorithmic decision making
+
+In [tools.md](tools.md) we have researched a few tools which we want to
```
Comment (anneschuth) [inline docs/Projects/TAD/tools_comparison.md:3]:
```suggestion
[We have researched](tools.md)  a few tools which we want to
```

File: docs/Projects/TAD/tools_comparison.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+investigate further, this document is the next step in that investigation.
+The checklist we will lay these tools against is written down at
+[index.md](index.md). The Fulfilled column will give a numerical value based
```
Comment (anneschuth) [inline docs/Projects/TAD/tools_comparison.md:6]:
```suggestion
We created a [checklist](index.md) to compare these tools against. The Fulfilled column will give a numerical value based
```

File: docs/Projects/TAD/aiverify_checklist.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
+# AI Verify
+
+Introduction is written here [tools.md](tools.md#ai-verify)
```
Comment (anneschuth) [inline docs/Projects/TAD/aiverify_checklist.md:3]:
```suggestion
See the [introduction](tools.md#ai-verify).
```

File: docs/Projects/TAD/tools_comparison.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
+# Comparison of tools for transparency of algorithmic decision making
+
+[We have researched](tools.md) a few tools which we want to investigate
+further, this document is the next step in that investigation.
+We created a [checklist](index.md) to compare these tools against.
+The Fulfilled column will give a numerical value based
+on the priority which can then be compared with the other tools per section.
+The scoring is as follows: {M:4, S:3, C:2, W:-1}.
```
Comment (ChristopherSpelt) [inline docs/Projects/TAD/tools_comparison.md:8]: Arguably, I am nitpicking now, but the description of the way how you score is not complete. Maybe something like:
```suggestion
The scoring is as follows. Each requirement is assigned a value between 0 and a maximum depending on the priority; the maxima are defined as follows {M:4, S:3, C:2, W:-1}.
```

**Review by ChristopherSpelt — COMMENTED** [review]
Looks good! Just one minor optional suggestion.

File: docs/Projects/TAD/tools_comparison.md (hunk 9 above)
Comment (laurensWe) [reply docs/Projects/TAD/tools_comparison.md]: I didn't completely take over your suggestion, but made it for sure more clear. Do you agree with the new one?

File: docs/Projects/TAD/tools_comparison.md (hunk 1 above)
Comment (laurensWe) [reply docs/Projects/TAD/tools_comparison.md]: I want to agree with you, but for me it just is too large of a table to have it all next to each other with the comments :S

File: docs/Projects/TAD/ai_assesment_tool_checklist.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
+# ALTAI
```
Comment (anneschuth) [inline docs/Projects/TAD/ai_assesment_tool_checklist.md:1]: can you nest these files a little deeper (in a "tools" directory or something)? 

And maybe we should shift all of this to a "comparison" directory anyway?

**Pull request merged**